src/app/subnets/views/SubnetDetails/SubnetStaticIPs/DeleteStaticIP/DeleteStaticIP.tsx

```diff
@@ -0,0 +1,22 @@
+import ModelActionForm from "@/app/base/components/ModelActionForm";
+import type { SetSidePanelContent } from "@/app/base/side-panel-context";
+
+type Props = {
+  setSidePanelContent: SetSidePanelContent;
+};
+const DeleteStaticIP = ({ setSidePanelContent }: Props) => {
```

Comment:
Part of acceptance criteria is making sure the form receives a correct MAC address OR IP + subnet ID which can be later passed to a submit function.